data/styles/outdoors/include/priorities_4_overlays.prio.txt

```diff
@@ -713,6 +713,7 @@ highway-path-tunnel                                 # pathtext z13- (also has li
 === 2820
 
 highway-steps                                       # pathtext z13- (also has line z12-)
+highway-ladder
```

Comment:
i think it's ok to have a high priority in outdoor mode - the mode is primarily for exploration (where you're much more likely to need to use a ladder)